code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -75,6 +75,10 @@
 			var/mob/M = A
 			if(M.key && !M.client)
 				continue
+		if(ishuman(A))
+			var/mob/living/carbon/human/H = A
+			if(H.paralysis)
```

Comment:
should check for unconsciousness instead

Reply:
From testing, checking paralysis works as intended, checking `sleeping` or `M.stat == UNCONCIOUS` just results in mobs beating the players to death.